fire-hose-management-frontend/src/components/Departments/DepartmentOverlay.jsx

```diff
@@ -1,158 +1,207 @@
 import * as i18nIsoCountries from "i18n-iso-countries";
-import React, { useState } from "react";
+import React, {useState} from "react";
 import Stack from "@mui/material/Stack";
-import { Button, Grid, MenuItem } from "@material-ui/core";
-import { CTextField } from "../shared/input/CTextField";
-import { CSelect } from "../shared/input/CSelect";
-import { ValidatorForm } from "react-material-ui-form-validator";
-import { Overlay } from "../shared/Overlay";
-import { makeStyles } from "@material-ui/core/styles";
+import {Button, Checkbox, FormControlLabel, Grid, MenuItem} from "@material-ui/core";
+import {CTextField} from "../shared/input/CTextField";
+import {CSelect} from "../shared/input/CSelect";
+import {ValidatorForm} from "react-material-ui-form-validator";
+import {Overlay} from "../shared/Overlay";
+import {makeStyles} from "@material-ui/core/styles";
 
-const useStyles = makeStyles( ( theme ) => ({
-  checkbox: {
-    display: "table-cell",
-    verticalAlign: "middle"
-  },
-  footer: {
-    paddingTop: "24px",
-    paddingBottom: "32px"
-  }
-}) );
+const useStyles = makeStyles((theme) => ({
+    checkbox: {
+        display: "table-cell",
+        verticalAlign: "middle"
+    },
+    footer: {
+        paddingTop: "24px",
+        paddingBottom: "32px"
+    }
+}));
 
 
-i18nIsoCountries.registerLocale( require( "i18n-iso-countries/langs/de.json" ) );
+i18nIsoCountries.registerLocale(require("i18n-iso-countries/langs/de.json"));
 
-const countries = Object.values( i18nIsoCountries.getNames( "de", { select: "official" } ) ).map( ( country, index ) =>
-  <MenuItem key={index} value={country}>{country}</MenuItem>
-)
+const countries = Object.values(i18nIsoCountries.getNames("de", {select: "official"})).map((country, index) =>
+    <MenuItem key={index} value={country}>{country}</MenuItem>)
 
+const genderMenuItems = [
+    <MenuItem key={0} value={"MALE"}>Männlich</MenuItem>,
+    <MenuItem key={1} value={"FEMALE"}>Weiblich</MenuItem>,
+    <MenuItem key={2} value={"OTHER"}>Divers</MenuItem>
```

Comment:
Codacy has a fix for the issue: [Replace `····<MenuItem·key={2}·value={"OTHER"}>Divers</MenuItem>` with `··<MenuItem·key={2}·value={"OTHER"}>⏎····Divers⏎··</MenuItem>,`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
  <MenuItem key={2} value={"OTHER"}>
```